shardingsphere-db-protocol/shardingsphere-db-protocol-mysql/src/main/java/org/apache/shardingsphere/db/protocol/mysql/packet/command/query/text/MySQLTextResultSetRowPacket.java

```diff
@@ -64,6 +65,8 @@ public void write(final MySQLPacketPayload payload) {
                     payload.writeStringLenenc(((BigDecimal) each).toPlainString());
                 } else if (each instanceof Boolean) {
                     payload.writeBytesLenenc((Boolean) each ? new byte[]{1} : new byte[]{0});
+                } else if (each instanceof LocalDateTime) {
+                    payload.writeStringLenenc(each.toString().replace("T", " "));
```

Comment:
Is it better to consider about SimpleDataFormat?

Reply:
I think, LocalDateTime is better than SimpleDateFormat. The each originally belong to LocalDateTime. So i  think it maybe needn't use SimpleDateFormat replaced.